docker/pdo_services_base.dockerfile

```diff
@@ -20,9 +20,9 @@ FROM pdo_base:${PDO_VERSION}
 ARG UBUNTU_VERSION=22.04
 ARG UBUNTU_NAME=jammy
 
-ARG SGX=2.22
-ARG OPENSSL=3.0.12
-ARG SGXSSL=3.0_Rev1
+ARG SGX=2.25
+ARG OPENSSL=3.0.14
+ARG SGXSSL=3.0_Rev4
 
```

Comment:
you'll need to rebase to get rid of these.

Reply:
There is a merge pr waiting for you, Sir!